src/components/Map.js

```diff
@@ -18,10 +20,32 @@ const Map = () => {
       <div className="map">
         <div className="map__inner">
           <GoogleMapReact
-            bootstrapURLKeys={{ key }}
-            defaultCenter={center}
+            bootstrapURLKeys={{
+              key: 'AIzaSyCbfV0IAdkkGv-9mmuAkUJNzCPPfGRO6v0',
+            }}
+            defaultCenter={centerCoords}
             defaultZoom={zoom}
-          />
+            yesIWantToUseGoogleMapApiInternals={true}
+            onGoogleApiLoaded={({ map, maps }) => handleApiLoaded(map, maps)}
+          >
+            {data && data.query.pages.length > 0 && loaded
```

Comment:
Good eye, @jssckbl . That error is probably happening because `query` is undefined above ^

Reply:
You all could use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to solve for that.
 `data?.query?.pages?.length`

Reply:
Thank you for spotting this! Will fix it